frontend/src/components/single-contract/SingleContractHeading.tsx

```diff
@@ -32,6 +34,7 @@ const SingleContractHeading: React.FC<ISingleContractHeadingProps> = (
             </h3>
           </div>
           <div className="mt-4 flex items-center justify-between sm:mt-0 sm:ml-6 sm:flex-shrink-0 sm:justify-start">
+            {state !== JobState.Reported && <OptionsDropdown />}
```

Comment:
Jobs can be reported only while in the Available or Started state.  Not sure if you want that check here or in the options dropdown, but it should probably be in one of them.